.github/workflows/validation.yaml

```diff
@@ -40,6 +40,12 @@ jobs:
 
       - name: setup carvel tooling binaries
         uses: vmware-tanzu/carvel-setup-action@v1
+        with:
+          only: ytt
+          ytt: v0.42.0 # Todo unlock as soon as 0.43 is working.
+
+      - run: |
```

Comment:
do you want to leave the version command in?

Reply:
I'll take it out as soon as 0.43 is working.

Reply:
(in a future pr)